spec/requests/v1/mods_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe "Mods export" do
```

Comment:
How is this test passing if the controller is using a `:mod` parameter which appears to be missing here?

Reply:
Rails magick ✨

Reply:
Rails assigns the body to a parameter derived from the controller name.

Reply:
TIL